docs/resources/server.md

```diff
@@ -6,26 +6,44 @@ description: |-
   Server resource schema. Must have a region specified in the provider configuration.
   ~> This resource is in beta and may be subject to breaking changes in the future. Use with caution. See our guide https://registry.terraform.io/providers/stackitcloud/stackit/latest/docs/guides/opting_into_beta_resources for how to opt-in to use beta resources.
   Example Usage
+  With key pair
+  ```terraform
+  resource "stackitkeypair" "keypair" {
+    name       = "example-key-pair"
+    publickey = chomp(file("path/to/idrsa.pub"))
+  }
+  resource "stackitserver" "user-data-from-file" {
+    projectid   = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+    bootvolume = {
+      size        = 64
+      sourcetype = "image"
+      sourceid   = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+    }
+    name         = "example-server"
+    machinetype = "g1.1"
+    keypairname = stackitkeypair.keypair.name
+    userdata    = file("${path.module}/cloud-init.yaml")
+  }
+  ```
   Boot from volume
-  
-  resource "stackit_server" "boot-from-volume" {
-    project_id = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+  ```terraform
+  resource "stackitserver" "boot-from-volume" {
```

Comment:
all resource names and field names have the wrong format in the generated md file. 

Should not it be stackit_server 

also following fields and resource names do not follow the snake format